webhooks-extension/src/components/WebhookBranches/WebhookBranches.scss

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import '~carbon-components/scss/globals/scss/vars';
+
+#webhook-branches-modal {
+  .bx--data-table tbody td.cellText {
+    padding-left: 22.5px;
+  }
+
+  .noBranches {
+    width: 100%;
+    text-align: center;
+    margin-top: 5px;
+    font-style: italic;
+    font-size: 1rem;
+  }
+
+  .bx--data-table tr th,
+  .bx--data-table tr th button {
+    background: #7190c1;
```

Comment:
is this consistent with other blues already in use?

Reply:
Yeah this is the exact same blue tone used for the webhooks table.